searchlib/src/vespa/searchlib/attribute/postinglistsearchcontext.h

```diff
@@ -113,8 +111,8 @@ class PostingListSearchContextT : public PostingListSearchContext
     static const long MIN_APPROXHITS_TO_NUMDOCS_RATIO_BEFORE_APPROXIMATION = 10;
 
     PostingListSearchContextT(const Dictionary &dictionary, uint32_t docIdLimit, uint64_t numValues,
-                              bool hasWeight, const PostingList &postingList, const IEnumStore &esb,
-                              uint32_t minBvCocFreq, bool useBitVector, const ISearchContext &baseSearchCtx);
+                              bool hasWeight, const PostingList &postingList, uint32_t minBvCocFreq,
```

Comment:
No need to keep old misspellings, consider using `minBvDocFreq`

Reply:
Fixed